RELEASING.md

```diff
@@ -12,9 +12,9 @@ as the last step, which publishes a snapshot build to
 
 ## Release cadence
 
-This repository roughly targets monthly minor releases from the `main` branch on the Friday after
+This repository roughly targets monthly minor releases from the `main` branch on the Tuesday after
```

Comment:
I'm thinking to change the Instrumentation release date to the Thursday after this